dnf5/include/dnf5/version.hpp

```diff
@@ -27,6 +27,7 @@ namespace dnf5 {
 /// Application version
 /// @since 5.0
 struct ApplicationVersion {
+    std::uint16_t prime;
```

Comment:
Could you document a meaning of the version fields here in the header file? People studying API reads the header files, not the commit messages.